clients/admin-ui/src/features/user-management/PermissionsForm.tsx

```diff
@@ -71,6 +75,14 @@ const PermissionsForm = () => {
     return <Spinner />;
   }
 
+  if (!isOwner && userPermissions?.roles?.includes(RoleRegistryEnum.OWNER)) {
```

Comment:
added this because we were able to demote an owner without it

![image](https://user-images.githubusercontent.com/24641006/225741856-b826d4e4-daa7-462c-b94a-0b9243109d7d.png)